config-overrides.js

```diff
@@ -14,29 +14,26 @@ module.exports = function override(config, env) {
           test: /[\\/]node_modules[\\/](ethers|@ethersproject)[\\/]/,
           name: 'ethers',
           chunks: 'all'
-        },
-        commons: {
-          test: /[\\/]node_modules[\\/]/,
-          // cacheGroupKey here is `commons` as the key of the cacheGroup
-          name(module, chunks, cacheGroupKey) {
-            const moduleFileName = module
-              .identifier()
-              .split('/')
-              .reduceRight((item) => item);
-            const allChunksNames = chunks.map((item) => item.name).join('~');
-            return `${cacheGroupKey}-${allChunksNames}-${moduleFileName}`;
-          },
-          chunks: 'all',
-        },
+        }
+        // commons: {
```

Comment:
what is the purpose of commenting this out?

Reply:
this config will separate to many chunks and load too many chunks at a same time make page slower (As I know Chrome has a limit of 6 connections per host name). So I commenting out it and will be test again in future.